python-manual/modules/ROOT/pages/bookmarks.adoc

```diff
@@ -72,18 +100,22 @@ def create_person(tx, name):
 # Create an employment relationship to a pre-existing company node.
 # This relies on the person first having been created.
 def employ(tx, person_name, company_name):
-    tx.run("MATCH (person:Person {name: $person_name}) "
-           "MATCH (company:Company {name: $company_name}) "
-           "CREATE (person)-[:WORKS_FOR]->(company)",
-           person_name=person_name, company_name=company_name)
+    tx.run("""
```

Comment:
Your points do make sense. Maybe it's just worth accepting the downsides for that. Because

> Maybe it is a case where we can improve the code side to make user experience smoother? 

We will not start manipulating Cypher queries in the driver. For what the driver is concerned, the query is some opaque string. The driver does and should not know what that sting means. So there is no way it would know how to reformat it (e.g., removing white space) without changing the semantics.